ice-adapter/src/main/java/com/faforever/iceadapter/gpgnet/GPGNetServer.java

```diff
@@ -46,7 +47,20 @@ public static void init() {
         }
 
         try {
-            serverSocket = new ServerSocket(IceAdapter.GPGNET_PORT);
+            for(int i = 0;;i++) {
+                try {
+                    serverSocket = new ServerSocket(IceAdapter.GPGNET_PORT);
```

Comment:
not sure how this works when used in static, but shouldn't this be closed at some point?

Reply:
yeah, I think it should, although I'm not sure if it's even open after failing to bind.

Reply:
well I mean, when application finishes, I don't see any close.